### PR TITLE
Chores form

### PR DESCRIPTION
**Pull request by mboneil10**
This was terrible. I still haven't made it so that a user can't enter whatever they want (validations and whatnot). Will do that next.

### CODE REVIEW

**Comment by mboneil10** [comment]
We still have to make the same things happen for events and finances.

**Comment by mboneil10** [comment]
Since everything is in place, I'll start writing events and finances forms. Edit: I actually want to wait until this is merged. I think it's enough to just have the chores form.

**Review by onocy — COMMENTED** [review]
This was a lot of work, nice job.

**Review by onocy — APPROVED** [review]

**Pull request merged**
